lib/core/sync_job_runner.rb

```diff
@@ -23,10 +23,16 @@ class SyncJobRunner
 
     def initialize(connector_settings)
       @connector_settings = connector_settings
-      @sink = Core::OutputSink::EsSink.new(connector_settings.index_name, @connector_settings.request_pipeline)
+      @ingester = Core::Ingestion::Ingester.new(Core::Ingestion::EsSink.new(connector_settings.index_name, @connector_settings.request_pipeline))
       @connector_class = Connectors::REGISTRY.connector_class(connector_settings.service_type)
       @sync_finished = false
       @sync_error = nil
+      @status = {
+        :indexed_document_count => 0,
+        :deleted_document_count => 0,
+        :indexed_document_volume => 0,
+        :error => nil
+      }
```

Comment:
I don't think we need this

Reply:
It was left here after a rebase, what's up with these lines?

Reply:
Previously we use `@status` because we want to `@status[:indexed_document_count] +=1` for every yielded document. But now, we will just use `@ingester.ingestion_stats.merge(:metadata => connector_instance.metadata)`.

I will make the change in my PR.